### DISCUSSION TITLE
Improve description of what flakeheaven is/does in README

### DISCUSSION THREAD

**Discussion by mcarans** [flakeheaven/flakeheaven#151] [Ideas]
A Google search on "flakeheaven" brings up [this reddit post](https://www.reddit.com/r/learnpython/comments/u5z0z2/what_problem_is_flakeheaven_solving/). 

"Hey there! I usually use flake8, pydocstyle, bandit, etc., as part of my linting before getting to my unit tests. Is Flakeheaven replacing these into one linter or does it allow for easy tunning in my toml file only? Not really sure what problem this library is solving. Thanks!"

One of the comments says:
"I feel your pain; I hate projects like this. Paragraph #1 in any readme of a public project should be a simple description of the purpose of the project and the problems it solves without any dependencies.

FlakeHeaven is a Flake8 wrapper to make it cool.

Ok... so now I have to look up what Flake8 is? And what makes it uncool?

So, options like --ignore and --select [are] unsupported.

Yep, not even a complete sentence. I especially like how they are defining the project in terms of what it can't do.

Read the documentation for more information.

Links to a page that is a literal copy of the readme.

/rant off"

While the criticisms are somewhat blunt, I don't think they should be dismissed. How could we make it clearer what flakeheaven is and does in the README?

**Comment by pwoolvett** [comment]
Agreed.

Summary should be self sufficient (I also dont like 'for humans' or 'smart' or 'cool' adjectives...)

Perhaps something like 'flakeheaven is a python linter [...] built around flake8 [...] <with x and y in mind/focusing on .../configurable in a single file[...] For nitpicky users>'

(At which point, if you've not heard of flake8 or linting or python, you're in the wrong place!)


Also, I'd love a flake logo!


The linked page is not a copy, the readme itself is part of the documentation...

**Reply by mcarans** [reply]
Yes I think something like what you've suggested would be good. Whoever reads the summary should be able to ascertain why they would want to use flakeheaven rather than using flake8 directly. In my case, the main attraction of flakeheaven is that it supports pyproject.toml (something that the author of flake8 unfortunately refuses to do because he dislikes the toml format).

**Reply by pwoolvett** [reply]
## flake8:
> "flake8 is a python tool that glues together pycodestyle, pyflakes, mccabe, and third-party plugins to check the style and quality of some python code. "

> Flake8 is a wrapper around these tools:
> 
>     PyFlakes
>     pycodestyle
>     Ned Batchelder's McCabe script

> Flake8: Your Tool For Style Guide Enforcement

## pyflakes:
> A simple program which checks Python source files for errors.


## flakeheaven
What do you think of 
> flakeheaven is a python linter built around flake8 to enable inheritable and complex toml configuration.

for the tagline? I think that pretty much covers the core of what it does, as the rest are nice-to haves but should work completely independent...What about refactoring out the pylint plugin and formatters to different packages? maybe not now, but because of this, I wouldn't mind not including those as part of the tagline.

We can go into more details later checking the rest of the readme if required...

**Reply by mcarans** [reply]
I think that is much clearer. A good tagline makes the rest of the documentation that much easier to follow.
